static/src/stylesheets/module/experiments/_update-account.scss

```diff
@@ -0,0 +1,18 @@
+.site-message--membership-action-required {
+    color: $garnett-neutral-5;
+    background: $garnett-neutral-1;
+    &:after {
+        @include multiline(3, $garnett-neutral-7, top);
+        content: '';
+        display: block;
+        height: 4px * 3; /*height of multiline (4px) times the number of lines (4)*/
+        margin-bottom: -1px;
+        width: 100%;
+    }
+    a {
+        color: $garnett-neutral-5;
+        font-weight: 900;
+        text-decoration: underline;
+    }
+
+}
```

Comment:
is there a particular reason only this one message will have a multiline on the bottom? Feels quite generic as in this could be applied to the cookie message too

Reply:
I asked for a design, and they gave me a multiline.